force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -269,19 +320,24 @@ public with sharing class ClassesObjectsMethods {
      * @return return a list of books.
      */
     public static List<Book> generateBookList(Book myBook1) {
-        // Create a list of books
-
-        // Update the title and author of myBook1
+     // Create a list of books
+    List<Book> books = new List<Book>();
 
-        // Update the pages for myBook1 to 352
+    // Update the title and author of myBook1
+    myBook1.title = 'Harry Potter and the Chamber of Secrets';
+    myBook1.author = 'J.K. Rowling';
 
-        // Create a new Book object with title "A Brief History of Time" and author "Stephen Hawking"
+    // Update the pages for myBook1 to 352
+    myBook1.pages = 352;
 
-        // Update the pages for A Brief History of Time to 256
+    // Create a new Book object with title "A Brief History of Time" and author "Stephen Hawking"
+    Book myBook2 = new Book('A Brief History of Time', 'Stephen Hawking');
 
-        // Add the book to the list
-        
-        return null; // Replace null with the variable you used to store the result      
+    // Add the book to the list
+    books.add(myBook1);
+    books.add(myBook2);
+    
```

Comment:
good work here. Another technique would be to add the results of the new book evaluation directly to the the List.